tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -0,0 +1,194 @@
+apiVersion: manila.openstack.org/v1beta1
+kind: manila
+metadata:
+   name: manila
+   namespace: openstack
+spec:
+   serviceUser: manila
+   customServiceConfig: |
+      [DEFAULT]
+      debug = true
+   databaseInstance: openstack
+   databaseUser: manila
+   rabbitMqClusterName: rabbitmq
+   manilaAPI:
+      replicas: 1
+      containerImage: quay.io/tripleozedcentos9/openstack-manila-api:current-tripleo
+   manilaScheduler:
+      replicas: 1
+      containerImage: quay.io/tripleozedcentos9/openstack-manila-scheduler:current-tripleo
+   manilaBackup:
+      replicas: 1
+      containerImage: quay.io/tripleozedcentos9/openstack-manila-backup:current-tripleo
+   manilaVolumes:
+      volume1:
+         containerImage: quay.io/tripleozedcentos9/openstack-manila-volume:current-tripleo
+         replicas: 1
```

Comment:
These `manilaBackup` and `manilaVolumes` blocks do not seem to exist in the actual `Manila` CRD.  Was this copy-pasted from perhaps `Cinder` KUTTL tests and then not modified?  It appears that way to me.

Reply:
Thanks for your feedback, resolved.